src/screens/auth/login/login.screen.tsx

```diff
@@ -1,15 +1,48 @@
 import React from 'react';
-import styled from 'styled-components';
+import { Text, Image, Input } from 'components/primitive';
+import { Header } from 'components/common/auth-header.component';
+import {
+  LoginContainer,
+  LoginFormContainer,
+  FormContainer,
+  LoginText,
+  StyledButton,
+  SocialLoginContainer,
+  TextContainer,
+  SocialIconsContainer,
+} from './login.screen.styles';
 
-const Section = styled.section`
-  height: 75vh;
-  display: flex;
-  margin-top: 9rem;
-  justify-content: center;
-`;
+export const LoginScreen = () => {
+  return (
+    <LoginContainer>
+      <Header />
 
-export function LoginScreen() {
-  return <Section>LOGIN</Section>;
-}
+      <LoginFormContainer>
+        <FormContainer>
+          <LoginText variant='contentHeader' center tx='auth.getStarted' />
+          <Input type='text' label='Enter your Email or DID' placeholder='hello@example.com' />
 
-export default LoginScreen;
+          <StyledButton
+            variant='primary'
+            label={{ tx: 'auth.login' }}
+            onClick={() => 'clicked'}
+            color='primaryGradient'
+          />
+
+          <SocialLoginContainer>
+            <TextContainer>
+              <Text variant='contentHeader' center tx='common.or' />
+              <Text variant='contentHeader' center tx='common.alternativeOption' />
+            </TextContainer>
+
+            <SocialIconsContainer>
+              <Image name='LoginWithGitHub' height={5} width={7} />
```

Comment:
These should be should have the functionality of `button`.

Reply:
yeah, Image supports `onClick ` event handler.

Reply:
Okay, then we need an optional prop to enable `cursor: pointer`.

Reply:
done. pushed in the latest commit.